portal-wire-protocol.md

```diff
@@ -124,11 +129,10 @@ selector     = 0x03
 nodes        = Container(total: uint8, enrs: List[ByteList[2048], max_length=32])
 ```
 
-* `total`: The total number of `Nodes` response messages being sent. Currently fixed to only 1 response message.
-* `enrs`: List of byte strings, each of which is an RLP encoded ENR record.
-    * Individual ENR records **MUST** correspond to one of the requested distances.
-    * It is invalid to return multiple ENR records for the same `node_id`.
-    * The ENR record of the requesting node **SHOULD** be filtered out of the list.
+- `total`: The total number of `Nodes` response messages being sent. Currently fixed to only 1 response message.
+- `enrs`: List of byte strings, each of which is an RLP encoded ENR record.
+  _ Individual ENR records **MUST** correspond to one of the requested distances.
+  _ It is invalid to return multiple ENR records for the same `node_id`. \* The ENR record of the requesting node **SHOULD** be filtered out of the list.
```

Comment:
This unordered list went wrong.

And to add on that, all the unordered list changes are not needed in the first place as asterisks, pluses, and hyphens may all be used for this in markdown.

The same applies for the emphasis asterisks vs underscores.